packages/engine/src/worker/runner/python/message.py

```diff
@@ -356,9 +371,21 @@ def user_errors_to_fbs_bytes(errors):
 
     UserErrors.Start(builder)
     UserErrors.AddInner(builder, vector_offset)
-    user_errors_offset = UserErrors.End(builder)
+    return UserErrors.End(builder)
+
+
+def outbound_user_errors_to_fbs_bytes(sim_id, errors):
+    # `initialSize` only affects performance (slightly), not correctness.
+    builder = flatbuffers.Builder(initialSize=len(errors))
+    errors_offset = user_errors_to_fbs(builder, errors)
 
-    builder.Finish(user_errors_offset)
+    RunnerOutboundMsg.Start(builder)
+    RunnerOutboundMsg.AddSimSid(builder, sim_id)
```

Comment:
I think we should make sim_id optional rather than trying to use 0 as an 'invalid' ID. That seems way too easy for us to make a mistake without getting warned about it.

Reply:
https://app.asana.com/0/1199548034582004/1201758381066811/f